pkg/migrations/sql/preinstall/000-utils.sql

```diff
@@ -13,11 +13,12 @@
 CREATE OR REPLACE PROCEDURE execute_everywhere(command_key text, command TEXT, transactional BOOLEAN = true)
```

Comment:
I'm not quite sure what this function does, but it scares the crap out of me...

Reply:
(from a perms/safety perspective) if it is safe, I'd say we need a comment about why it is safe etc. And that table needs to have write permissions revoked from the writer user most definitely, cause otherwise, the writer could execute arbitrary stuff on new data nodes)